ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -2,7 +2,7 @@ base_images:
   golangci-lint:
     name: golangci-lint
     namespace: ci
-    tag: v1.37.1
+    tag: v1.43.0
```

Comment:
You can only use the image *after* it is present in the cluster.

Reply:
Thanks, I will split the PR then.